test/testfiles/test_cube_unstructured_1.inp

```diff
@@ -0,0 +1,250 @@
+*Heading
+ C:\Users\Erik\Downloads\gmsh-4.8.4-Windows64\cube_unstructured_1.inp
```

Comment:
Is it necessary to have this heading in the file? We should not include filepaths.

Reply:
Removed. Done.